includes/3rd-party/divi.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Adds additional compatibility with Divi.
+ *
+ * @package 3rd-Party
+ */
+
+/**
+ * It disables Yoast initialization when running the Divi preview in the editor.
+ * It avoids an existing conflict that prevents the preview from loading.
+ */
+function sensei_fix_divi_yoast_conflict() {
+	if ( isset( $_GET['et_block_layout_preview'] ) && '1' === $_GET['et_block_layout_preview'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
The ideal would be to also have a check to run it only in lessons with Learning Mode, but it's too early to check this.